public/privacy.html

```diff
@@ -0,0 +1,1142 @@
+                                </path>
+                            </svg></button></span><button type="button" class="ant-btn ml-2 mr-2"><span>登
+                            入</span></button><button type="button" class="ant-btn ant-btn-link ant-btn-icon-only"><i
+                            aria-label="icon: menu" class="anticon anticon-menu"><svg viewBox="64 64 896 896"
+                                focusable="false" class="" data-icon="menu" width="1em" height="1em" fill="currentColor"
+                                aria-hidden="true">
+                                <path
+                                    d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z">
+                                </path>
+                            </svg></i></button>
+                </div>
+            </header>
+            <main class="sc-oTmHo fcqzIr ant-layout-content" id="layout-content">
+                <div class="sc-pbJYR hyPNMc">
+                    <section class="sc-qZuGl cezain">
+                        <h1 class="ant-typography sc-pjGbq cpFNgU">使用條款</h1>
+                        <div class="container">
+                            <div class="ant-card sc-qQXoI XBhvs ant-card-bordered">
+                                <div class="ant-card-body">
+                                    <h2 class="ant-typography sc-pjGbq cpFNgU">隱私權條款</h2>
+                                    <h3 class="ant-typography sc-psdsT dQpeAq">隱私權保護政策的適用範圍</h3>
+                                    <p>隱私權保護政策內容，包括 {{}} 如何處理在您使用網站服務時收集到的個人識別資料。隱私權保護政策不適用於 {{}}
+                                        以外的相關連結網站，也不適用於非 {{}} 所委託或參與管理的人員。</p>
+                                    <h3 class="ant-typography sc-psdsT dQpeAq">個人資料的蒐集、處理及利用方式</h3>
+                                    <p>當您造訪 {{}} 或使用 {{}}
+                                        所提供之功能服務時，我們將視該服務功能性質，請您提供必要的個人資料，並在該特定目的範圍內處理及利用您的個人資料；非經您書面同意， {{}}
+                                        不會將個人資料用於其他用途。</p>
+                                    <p>於一般瀏覽時，伺服器會自行記錄相關行徑，包括您使用連線設備的IP位址、使用時間、使用的瀏覽器、瀏覽及點選資料記錄等，做為我們增進網站服務的參考依據，此記錄為內部應用，決不對外公佈。
+                                    </p>
+                                    <h3 class="ant-typography sc-psdsT dQpeAq">網站對外的相關連結</h3>
+                                    <p>{{}} 的網頁提供其他網站的網路連結，您也可經由 {{}} 所提供的連結，點選進入其他網站。但該連結網站不適用 {{}}
```

Comment:
What value is supposed to be filled in this `{{}}`? 
It seems to be empty, and there are also some other places like this, please check.

Reply:
updated